convex/schema.ts

```diff
@@ -33,6 +33,12 @@ const profile = defineTable({
   role: v.optional(v.union(v.literal("admin"), v.literal("user"))),
 }).index("occupation", ["occupation"]);
 
+//Schema for stats
```

Comment:
Why do we need this?

Reply:
Do you mean the schema for stats or the role ??

Reply:
Yes

Reply:
It's used to record every time a user scan's thier QR code. Which is then used to calculate the total attendance and if they are eligible for a free day